containers/flatpak/prepare

```diff
@@ -0,0 +1,69 @@
+#!/bin/sh
+
+set -eu
+[ $(basename -- "$0") = "prepare" ] && cd "$(realpath -m "$0"/../../..)"
+
+FLATPAK_ID="${1:-org.cockpit_project.CockpitClient}"
+
+if test "$#" -le 1; then
+    if ! test -f ./configure; then
+        ./autogen.sh
```

Comment:
Could/should we give the indended arguments here with the --disable-* and the mock env bits? It's a bit sad that this needs to be repeated between here and the yml.in, but we could at least remove it from .github/workflows/flatpak-test.yml then (it really doesn't belong there)

Reply:
Discussed in person.